hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/state/WrappedWritableKVStateTest.java

```diff
@@ -57,4 +60,69 @@ void commitGoesToDelegateNotBackingStore() {
         assertThat(delegate.get(B_KEY)).isEqualTo(BLACKBERRY); // Has the new value
         assertThat(delegate.get(E_KEY)).isEqualTo(ELDERBERRY); // Has the new value
     }
+
+    @Nested
+    @DisplayName("size")
```

Comment:
"size" is implied by the name of the class, `SizeTest`. What value does this display name add to the code?

Reply:
Followed the same pattern in the current test class , where there is a nested test class for each method in the class. And that looked very nice.